### ISSUE TITLE
all books (epub) fail to De-DRM

### ISSUE THREAD

**Issue by AlfredoWilliam** [apprenticeharper/DeDRM_tools#1663]
Hi, as mentioned above, all my books fail to De-DRM. They were copied from my Laptop to the PC.
System: Win10 20H2 Build 19042.906  64bit
DeDRM: 7.2.1
Calibre: 5.14 64bit
Thanks in advance for any advise.
[dedrmlog2.txt](https://github.com/apprenticeharper/DeDRM_tools/files/6294925/dedrmlog2.txt)

**Comment by ElleKayEm** [comment]
If you downloaded them on a different computer, do you also have Adobe Digital Editions authorized with your Adobe ID on the same machine and user as calibre with DeDRM?

**Comment by AlfredoWilliam** [comment]
No, it is a long time ago that I downloaded it, and I do not remember the old Adobe ID...

**Comment by ElleKayEm** [comment]
That's not going to work then.  DeDRM needs to extract the key associated with your Adobe ID.  So if you can't open them with ADE, DeDRM can't unlock them either.

**Comment by AlfredoWilliam** [comment]
Thank you again. Ok, I did not know that. So I will have to remember the original ID.

**Comment by ElleKayEm** [comment]
Here's a page about resetting your password and also help if you no longer have access to the email address:  https://helpx.adobe.com/manage-account/using/change-or-reset-password.html
